include/mbedtls/dhm.h

```diff
@@ -1,7 +1,12 @@
 /**
  * \file dhm.h
  *
- * \brief Diffie-Hellman-Merkle key exchange.
+ * \brief This file contains DHM definitions and functions.
```

Comment:
Whatever wording we decide on in https://github.com/ARMmbed/mbedtls/pull/1513#discussion_r180687267, we should use an analogous one here.

Reply:
Is "This file provides an API for the DHM key exchange." accurate?

Reply:
I'd be fine with that. @gilles-peskine-arm ?

Reply:
done